src/style.css

```
@@ -1,3 +1,85 @@
 body {
+  width: 100vw;
+  margin: 0 auto;
+  background: lightgray 50%;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
```

These are default values, will be better if you remove it everywhere
```suggestion
  font-weight: 500;
```